drivers/ng_at86rf2xx/ng_at86rf2xx_getset.c

```diff
@@ -235,6 +235,20 @@ void ng_at86rf2xx_set_txpower(ng_at86rf2xx_t *dev, int16_t txpower)
 #endif
 }
 
+uint8_t ng_at86rf2xx_get_max_retries(ng_at86rf2xx_t *dev)
+{
+    return (ng_at86rf2xx_reg_read(dev, NG_AT86RF2XX_REG__XAH_CTRL_0) >> 4);
+}
+
+void ng_at86rf2xx_set_max_retries(ng_at86rf2xx_t *dev, uint8_t max)
+{
+    max = (max > 7) ? 7 : max;
+    uint8_t tmp = ng_at86rf2xx_reg_read(dev, NG_AT86RF2XX_REG__XAH_CTRL_0);
+    tmp = (tmp & (NG_AT86RF2XX_XAH_CTRL_0__MAX_CSMA_RETRIES |
+                  NG_AT86RF2XX_XAH_CTRL_0__SLOTTED_OPERATION)) | (max << 4);
```

Comment:
Just out of curiosity, why don't we use SLOTTED_OPERATION?
Why don't we let the transceiver acknowledge automatically?